koku/masu/database/trino_sql/openshift/cost_model/monthly_cost_gpu.sql

```diff
@@ -0,0 +1,89 @@
+INSERT INTO postgres.{{schema | sqlsafe}}.reporting_ocpusagelineitem_daily_summary (
+    uuid,
+    report_period_id,
+    cluster_id,
+    cluster_alias,
+    data_source,
+    usage_start,
+    usage_end,
+    namespace,
+    node,
+    resource_id,
+    pod_labels,
+    all_labels,
+    source_uuid,
+    cost_model_rate_type,
+    cost_model_gpu_cost,
+    monthly_cost_type,
+    cost_category_id
+)
+SELECT
+    uuid() as uuid,
+    {{report_period_id}} as report_period_id,
+    CAST({{cluster_id}} AS varchar(50)) as cluster_id,
```

Comment:
monthly_project_tag_based.sql

```
    {{cluster_id}} as cluster_id,
    {{cluster_alias}} as cluster_alias,
```

It may be good for you to reevaluate which cast are necessary here.